src/mobile-token/MobileTokenContext.tsx

```diff
@@ -97,7 +100,11 @@ export const MobileTokenContextProvider: React.FC = ({children}) => {
   const mobileTokenEnabled = hasEnabledMobileToken();
 
   const enabled =
-    mobileTokenEnabled && userId && authStatus === 'authenticated';
+    mobileTokenEnabled && !!userId && authStatus === 'authenticated';
```

Comment:
Is the `!!userId` change intentional here? If so, why is that done? 🤔

Reply:
To make it typed as `boolean`.
![Screenshot 2024-03-12 at 15 24 14](https://github.com/AtB-AS/mittatb-app/assets/675421/73679b58-840d-4658-a83c-75b28f2b8b7c)